### DISCUSSION TITLE
Twin.macro + NextJs + Typescript

### DISCUSSION THREAD

**Discussion by Miodrag** [ben-rogerson/twin.macro#629] [Q&A]
Hello Ben, 

Is it possible to have twin.macro with nextjs + ts or it must be with emotion?

I'm getting Module '"twin.macro"' has no exported member 'styled'. Did you mean to use 'import styled from "twin.macro"' instead?

**Comment by ben-rogerson** [comment]
Hey there, twin works with a css styling library like emotion or styled-components. You'll be able to use typescript with either library.
